config/input.ron

```diff
@@ -1,16 +1,16 @@
 (
     axes: {
         "move_x": Emulated(
-            pos: Key(D),
-            neg: Key(A),
+            pos: Key(F),
```

Comment:
why are the keys shifted over?

Reply:
It's my personal preference (I use home row for gaming). I could switch it back match the mainstream WASD if you want. The starter project doesn't do anything with the input anyways.